src/components/help/help.stories.mdx

```diff
@@ -51,6 +51,21 @@ Hover over the help component to see the different positions.
   </Story>
 </Preview>
 
+### Help with Tooltip color overrides
+The tooltip background and font colors can be overridden using the `tooltipBgColor` and `tooltipFontColor` props. These 
+props accept and valid css color value.
+
+<Preview>
+  <Story name="with Tooltip color overrides">
```

Comment:
this story should be ignored in `chromatic` the same as `default` and `with icons`